### ISSUE TITLE
EngineCrash 1354 at System/Platform/Misc.cpp:768

### ISSUE THREAD

**Issue by p2004a** [beyond-all-reason/spring#568]
**URL of infolog.txt**
gdb session: https://0x0.st/oRGH.txt

Even though it's a segfault, infolog itself is useless: https://0x0.st/oRlJ.log + https://0x0.st/oRlJ.log

**Engine Version**
105.1.1-1354-g72b2d55 bar

**GDB Stack Trace**

```
#0  Platform::GetRawMacAddr () at /spring/rts/System/Platform/Misc.cpp:768
#1  0x0000555555f19d11 in Platform::GetMacAddrHash[abi:cxx11]() () at /spring/rts/System/Platform/Misc.cpp:486
#2  0x0000555555ac3398 in LuaConstPlatform::PushEntries (L=L@entry=0x7fff98074014) at /spring/rts/Lua/LuaConstPlatform.cpp:78
#3  0x0000555555ad2fde in CLuaHandle::AddEntriesToTable (L=0x7fff98074014, name=name@entry=0x5555569611f1 "Platform", entriesFunc=0x555555ac17f0 <LuaConstPlatform::PushEntries(lua_State*)>) at /spring/rts/Lua/LuaHandle.cpp:201
#4  0x0000555555b07009 in CLuaMenu::CLuaMenu (this=0x55559d4c9c20) at /spring/rts/Lua/LuaMenu.cpp:111
#5  0x0000555555b07235 in CLuaMenu::LoadHandler () at /spring/rts/Lua/LuaMenu.cpp:37
#6  0x0000555555eab370 in CLuaMenu::LoadFreeHandler () at /spring/rts/Lua/LuaMenu.h:29
#7  CLuaMenuController::CLuaMenuController (this=this@entry=0x55559d467470, menuName=...) at /spring/rts/Menu/LuaMenuController.cpp:33
#8  0x0000555555f24d48 in SpringApp::Startup (this=0x7fffffffdf90) at /spring/rts/System/SpringApp.cpp:647
#9  0x0000555555f25dbd in SpringApp::Init (this=0x7fffffffdf90) at /spring/rts/System/SpringApp.cpp:287
#10 0x0000555555f2600e in SpringApp::Run (this=this@entry=0x7fffffffdf90) at /spring/rts/System/SpringApp.cpp:848
#11 0x0000555555f093d3 in Run (argc=6, argv=0x7fffffffe0f8) at /spring/rts/System/Main.cpp:48
#12 0x00007ffff77b8290 in ?? () from /usr/lib/libc.so.6
#13 0x00007ffff77b834a in __libc_start_main () from /usr/lib/libc.so.6
#14 0x00005555558a39aa in _start ()
```

**Additional context**
Support thread: https://discord.com/channels/549281623154229250/1060475779080785970

The main thing is, the user was using VPN (WireGuard) and in this case, the interface didn't have address assigned. We might be able to reproduce it reliably with network namespaces probably.

**Comment by sprunk** [comment]
Can be assumed fixed?

**Comment by lhog** [comment]
Hopefully, no real way to test without a special setup

**Comment by sprunk** [comment]
Assuming fixed by 606dece8d89450eaee9da5713959b4544ae573f8 unless there's evidence to prove otherwise
